<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>staffReg</class>
 <widget class="QWidget" name="staffReg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>520</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>720</width>
    <height>520</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>720</width>
    <height>520</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="labTime">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>20</y>
     <width>300</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(0,0,0);
border: 2px solid #000000;background-color: rgb(255, 255, 255);
border-radius:5px;
min-width:80px</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>68</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>当前时间：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>68</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>消息通知：</string>
   </property>
  </widget>
  <widget class="QLabel" name="labInform">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>60</y>
     <width>300</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(0,0,0);
border: 2px solid #000000;background-color: rgb(255, 255, 255);
border-radius:5px;
min-width:80px</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labCam">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>380</width>
     <height>380</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>30</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>员工信息</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>110</y>
     <width>70</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;宋体&quot;;</string>
   </property>
   <property name="text">
    <string>工号：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>160</y>
     <width>70</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;宋体&quot;;</string>
   </property>
   <property name="text">
    <string>姓名：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>310</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;宋体&quot;;</string>
   </property>
   <property name="text">
    <string>签到时间：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>260</y>
     <width>70</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;宋体&quot;;</string>
   </property>
   <property name="text">
    <string>部门：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>210</y>
     <width>70</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;宋体&quot;;</string>
   </property>
   <property name="text">
    <string>性别：</string>
   </property>
  </widget>
  <widget class="QLabel" name="labSno">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>110</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(0,0,0);
border: 2px solid #000000;
background-color: rgb(255, 255, 255);
border-radius:5px;
min-width:80px</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labSname">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>160</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(0,0,0);
border: 2px solid #000000;background-color: rgb(255, 255, 255);
border-radius:5px;
min-width:80px</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labSsex">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>210</y>
     <width>31</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(0,0,0);
border: 2px solid #000000;background-color: rgb(255, 255, 255);
border-radius:5px;
min-width:8px</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labDepart">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>260</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(0,0,0);
border: 2px solid #000000;background-color: rgb(255, 255, 255);
border-radius:5px;
min-width:80px</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labRegTime">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>310</y>
     <width>91</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(0,0,0);
border: 2px solid #000000;background-color: rgb(255, 255, 255);
border-radius:5px;
min-width:80px</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnStart">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>390</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
		color:rgb(255, 255, 255);
		border: 2px solid #55aaff;
		border-radius:15px;
		background-color:rgb(58, 170, 255);
		min-width:80px
}
QPushButton:hover{ 
background-color: rgb(0, 52, 150);
border:2px solid #55aaff;
border-radius:15px;
}
QPushButton:pressed{
color:rgb(0, 0, 0);
background-color: rgb(58, 170, 255);
border:2px solid #55aaff;
border-radius:15px;
}
</string>
   </property>
   <property name="text">
    <string>开始签到</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnQuery">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>450</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
		color:rgb(0,0,0);
		border: 2px solid #b0b0b0;
background-color: rgb(255, 255, 255);
		border-radius:15px;
		min-width:80px
}QPushButton:hover{ 
color:rgb(255, 255, 255);
background-color: rgb(200, 200, 200);
border:2px solid  #b0b0b0;
border-radius:15px;
}
QPushButton:pressed{
color:rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
border:2px solid  #b0b0b0;
border-radius:15px;
}</string>
   </property>
   <property name="text">
    <string>查询签到</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOutUser">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>450</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
		color:rgb(0,0,0);
		border: 2px solid #b0b0b0;
background-color: rgb(255, 255, 255);
		border-radius:15px;
		min-width:80px
}QPushButton:hover{ 
color:rgb(255, 255, 255);
background-color: rgb(200, 200, 200);
border:2px solid  #b0b0b0;
border-radius:15px;
}
QPushButton:pressed{
color:rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
border:2px solid  #b0b0b0;
border-radius:15px;
}</string>
   </property>
   <property name="text">
    <string>退出帐号</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCloseCam">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>390</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
		color:rgb(255, 255, 255);
		border: 2px solid #c60e46;
		border-radius:15px;
		background-color: rgb(170, 0, 0);
		min-width:80px
}
QPushButton:hover{ 
background-color: rgb(140, 0, 0);
border:2px solid #c60e46;
border-radius:15px;
}
QPushButton:pressed{
color:rgb(0, 0, 0);
background-color: rgb(170, 0, 0);
border:2px solid #c60e46;
border-radius:15px;
}</string>
   </property>
   <property name="text">
    <string>关闭摄像头</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
